### PR TITLE
doc: update CLI help doc for notation sign and verify in RC.1

### LINKED ISSUE #383: Add examples for notation command help doc
We need to add more examples in `--help` information for each command in Alpha.4. These examples will be straightforward and easy to understand the typical usage of each command line with its flags.

Users can easily find these references using the `--help` for each `notation` command line and its flag. This will be a good foundation for further updates. 

- [ ] notation sign
- [ ] notation verify
- [ ] notation key
- [ ] notation certificate
- [ ] notation plugin

### PR DESCRIPTION
**Pull request by FeynmanZhou**
fix #383

### CODE REVIEW

File: cmd/notation/sign.go [hunk 1]
```diff
@@ -31,17 +31,23 @@ func signCommand(opts *signOpts) *cobra.Command {
...
 
-Example - Sign a container image using the specified key name
-  notation sign --key <key_name> <registry>/<repository>:<tag>
+Example - Sign an OCI artifact using a specified key
+  notation sign --key <key_name> <registry>/<repository>@<digest>
 
-Example - Sign a container image using the image digest
-  notation sign <registry>/<repository>@<digest>
+Example - Sign an OCI artifact using a local key and certificate file directly
+  notation sign --key-file <key_path> --cert-file <cert_path> <registry>/<repository>@<digest>
```
Comment (yizha1) [inline cmd/notation/sign.go:44]: This example should be removed, since we don't support sign using local key.

File: cmd/notation/verify.go [hunk 2]
```diff
@@ -25,9 +25,18 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
...
+		Use:   "verify [reference]",
+		Short: "Verify OCI artifacts",
+		Long: `Verify OCI artifacts
+
+Prerequisite: added a certificate into trust store and created a trust policy. 
+
+Example - Verify a signature on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
+  notation verify <registry>/<repository>:<tag>
+
+Example - Verify a signature on an OCI artifact identified by a digest:
```
Comment (yizha1) [inline cmd/notation/verify.go:37]: suggest move `digest` example to the first one, since it is the recommended way.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by patrickzheng200 — APPROVED** [review]
LGTM

**Review by priteshbandi — APPROVED** [review]
LGTM

**Pull request merged**